x-pack/legacy/plugins/ml/public/datavisualizer/file_based/breadcrumbs.js

```diff
@@ -5,14 +5,16 @@
  */
 
 
-import { ML_BREADCRUMB } from '../../breadcrumbs';
+import { ML_BREADCRUMB, ANOMALY_DETECTION_BREADCRUMB, DATA_VISUALIZER_BREADCRUMB } from '../../breadcrumbs';
 
 
 export function getFileDataVisualizerBreadcrumbs() {
   // Whilst top level nav menu with tabs remains,
   // use root ML breadcrumb.
   return [
-    ML_BREADCRUMB
+    ML_BREADCRUMB,
+    ANOMALY_DETECTION_BREADCRUMB,
```

Comment:
As above, we shouldn't have Anomaly Detection in here. Should just be Machine Learning / Data Visualizer I think. Or even Machine Learning / Data Visualizer / File based? 

![image](https://user-images.githubusercontent.com/7405507/64692153-5189b500-d48c-11e9-8971-74c58a973422.png)

Reply:
Yes, this was a mistake. Thank you for catching it 😄